dist/profile/manifests/buildmaster.pp

```diff
@@ -356,6 +363,13 @@
         aliasmatch => '(.*)/api/(json|python)(/|$)(.*)',
         path       => "${docroot}/empty.json",
       },
+      {
+        # Send all api/xml to `empty.xml` to prevent abusive clients
+        # (checkman) from receiving an invalid XML response and repeatedly attempting
```

Comment:
Presumably it is not checkman (whatever that is) that would be asking for XML.

Reply:
@jglick https://github.com/cppforlife/checkman/issues/74 (but it requests JSON)